src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -60,7 +61,72 @@ void *rom_data_lookup(uint32_t code);
  */
 bool rom_funcs_lookup(uint32_t *table, unsigned int count);
 
-typedef void __attribute__((noreturn)) (*reset_usb_boot_fn)(uint32_t, uint32_t);
+// The following is a list of bootrom functions available via rom_func_lookup and their signature
```

Comment:
Should this list also include `rom_table_code('S', 'D')` and `rom_table_code('S', 'F')` from https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/pico_double/double_init_rom.c ? (and `src/rp2_common/pico_float/float_init_rom.c`)

Reply:
nah, those aren't for public consumption